### ISSUE TITLE
Fill in the unit test of implemented bif

### ISSUE THREAD

**Issue by RyanSiu1995** [archseer/enigma#26]
Some of the unit tests are not implemented yet.
Required someone to fill them up.
- [ ] Math functions
- [ ] List functions
- [ ] Chrono functions

Here are the incomplete unit tests (without testing the exception case).
- [ ] erlang functions

**Comment by archseer** [comment]
Agreed! I think it's important to have some coverage, but in the future, we'll also cover some part of that via tests inside the runtime

**Comment by archseer** [comment]
I'm also going to set up code coverage reporting
